shared/src/metabase/mbql/schema.cljc

```diff
@@ -78,6 +78,12 @@
    (apply s/enum datetime-bucketing-units)
    "datetime-bucketing-unit"))
 
+(def DateExtractUnits
+  "Valid units to extract from a datetime."
+  (s/named
+    (apply s/enum #{:second :minute :hour :day :day-of-week :week :month :quarter :year})
+    "temporal-extract-units"))
```

Comment:
I wonder why the var name `DateExtractUnits` is different from the schema name `temporal-extract-units`.

Reply:
ah, I missed this when `s/date/temporal` the whole thing. Will change.